common.go

```diff
@@ -0,0 +1,25 @@
+// Common helpers to both the client and server
+
+package main
+
+import (
+	"fmt"
+	"rand"
+	"time"
+)
+
+func NewDeployId() string {
+	t := time.Now().UTC()
+	rand.seed(t.UnixNano())
```

Comment:
Yeah, otherwise anywhere else we use randomness we would have to seed as well, otherwise we wouldn't be sure that it's been seeded yet.